### DISCUSSION TITLE
A Simpler UI

### DISCUSSION THREAD

**Discussion by Skubed007** [godotengine/godot-proposals#9023] [GUI]
### Describe the project you are working on

flappy bird

### Describe the problem or limitation you are having in your project

Bad UI

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

Am new to godot and trying to learn it i absolutely love it but the UI seems a bit old and not good if the ui can be worked on it would be really nice make it look more modern like macos like feeling

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

more asthetic look a nicer UI plzzz

### If this enhancement will not be used often, can it be worked around with a few lines of script?

ig it can but no this should built in

### Is there a reason why this should be core and not an add-on in the asset library?

cause it is very very essential

**Comment by AThousandShips** [comment]
Converted to an issue as there's no details here on *how* to achieve this, which is required

**Comment by theraot** [comment]
From the little description, I think this is probably addressed by themes.

You can look up tutorials on making themes for Godot, stating - of course - with the official documentation: [Using the theme editor](https://docs.godotengine.org/en/stable/tutorials/ui/gui_using_theme_editor.html). If you do not know where to find tutorials, there is a short list on the official documentation: [Tutorials and resources](https://docs.godotengine.org/en/stable/community/tutorials.html), and of course, your prefered search engine should be able to help you find more.

On a similar note, if you can't or rather not create your own theme, there are some free Godot themes you can find online.

Are Godot themes for the editor or the games? Yes.

You might find Godot themes designer for one use or the other in mind, but they all can be used in your games or applied to the editor via editor settings.

If you have more specific questions about how to do some UI design, you might be able to find help in other sites, starting - of course - with the [official community sites](https://godotengine.org/community/).

And if you have more concrete ideas of how to improve Godot, those are welcome here. Post bugs and other problems at: https://github.com/godotengine/godot/issues

**Comment by Calinou** [comment]
You may be interested in https://github.com/passivestar/godot-minimal-theme :slightly_smiling_face:

**Comment by ismailivanov** [comment]
https://github.com/passivestar/godot-minimal-theme This theme should be the default for the engine. I can say that it increases the workspace and looks nice to the eye.
